tsconfig.json

```diff
@@ -15,7 +15,7 @@
 		"esModuleInterop": true,
 		"skipLibCheck": false,
 		"module": "esnext",
-		"typeRoots": [ "**/node_modules/@types" ],
+		"typeRoots": [ "./node_modules/@types", "packages/**/node_modules/@types" ],
```

Comment:
@samueljseay it couldn't find the `@types/jest` when I installed it, my guess was that `**/node_modules/@types` didn't match the root `node_modules`, not sure why, but the above seemed to fix it.

Reply:
Seems fine to me :) if it works it works 👍